### ISSUE TITLE
Sensors error

### ISSUE THREAD

**Issue by rahanozturk2** [lukasberbuer/weatherlink-live-local-python#2]
Hello, I use the library to get data from the internet, but after a couple of data it shows an error; KeyError: 'sensors'

**Comment by lukasberbuer** [comment]
How do you use the library to fetch data from the internet? The library is intended to be used with a local [Davis WeatherLink Live station](https://www.davisinstruments.com/weatherlinklive/)  + connected sensors.

Could you please report the JSON data, which leads to the parsing error?

**Comment by rahanozturk2** [comment]
Oh, I asked weatherlink discord channel and they link here. I have two different codes; one on the internet and the other one with cable. Your code matches with the cable one.

`parms = "api-key" + myAPI + "t" + str(int(time.time()))
    myTime = int(time.time())
    weatMsg = "api-key" + myAPI + "station-id" + str(stID) + "t" + str(int(time.time()))
    weatH = hmac.new(byte_key, weatMsg.encode(), hashlib.sha256).hexdigest()
    weatD = requests.get("https://api.weatherlink.com/v2/current/"
                       + str(stID)
                       +"?api-key=" + myAPI
                       + "&t="+ str(int(time.time()))
                       + "&api-signature="  + weatH
                      )
    weatJSON = weatD.json()["sensors"][0]["data"][0]`

so basically weatD.json() does not have sensors data. I found the same issue on the internet and they said it's about the available data transfer per hour/min etc.

**Comment by lukasberbuer** [comment]
Interesting would be the actual API response (`weatD.status_code`, `weatD.headers` and `weatD.text`). Could you show those please?

**Comment by rahanozturk2** [comment]
I found a similar code that produce the same error, maybe this link would help; [https://github.com/brandonbeasley/py_weatherlink2/issues/2](url)

**Comment by rahanozturk2** [comment]
weatD.status_code; `429`
weatD.headers; 
`{'Date': 'Wed, 29 Jun 2022 13:59:33 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '37', 'Connection': 'keep-alive', 'Retry-After': '27', 'X-RateLimit-Limit-Second': '10', 'X-RateLimit-Remaining-Second': '10', 'X-RateLimit-Limit-Hour': '300', 'RateLimit-Limit': '300', 'X-RateLimit-Remaining-Hour': '0', 'RateLimit-Remaining': '0', 'RateLimit-Reset': '27', 'X-Kong-Response-Latency': '2', 'Server': 'kong/2.0.2'}`
weatD.text; `'{"message":"API rate limit exceeded"}'`

**Comment by lukasberbuer** [comment]
Ok this seems to be out of scope of this library. You should definately check the `status_code` in your API calls before trying to parse the (expected) JSON message.

**Comment by rahanozturk2** [comment]
I found the problem, i set time.sleep to 10 secs on 5 mins data ((: Anyway, thank you
